src/components/LocationFormFragment.tsx

```diff
@@ -0,0 +1,56 @@
+import { Form, Input } from 'antd';
+import React from 'react';
+
+// TODO: use this fragment in signup form, note that "zip" was renamed to "zipCode"
+const LocationFormFragment: React.FC = () => {
+  return (
+    <>
+      <Form.Item
+        name="address"
+        label="Address"
+        className="stacked-inputs"
+        rules={[{ required: true, message: 'Please input your address' }]}
+      >
+        <Input placeholder="Address" />
+      </Form.Item>
+      <Form.Item
+        name="city"
+        label="City"
+        className="inline-block-third"
+        rules={[{ required: true, message: 'Please input city' }]}
```

Comment:
This could be limited to strings only

Reply:
Because the type of this input isnt numeric it should always be a string